src/site/content/en/learn/index.njk

```diff
@@ -1,68 +1,81 @@
 ---
-layout: default
+layout: default-next
 title: Learn
 description: |
   Learn best practices for the modern web and hone your skills with hands-on
   codelabs.
-date: 2018-11-05
+date: 2021-10-29
 pageScripts:
   - '/js/learn.js'
 ---
-{# *-landing-page classes are a holdover from devsite. #}
-{# It would be great to remove them as part of the v1 migration. #}
-<div class="learn-landing-page">
+{% from 'macros/icon.njk' import icon with context %}
 
-  <div class="w-layout-container">
-    <header class="w-page-header">
-      <h1 class="w-page-header__headline">{{ title }}</h1>
-      <p class="w-page-header__copy">
-        {{ 'i18n.learn.header' | i18n(locale) }}
-      </p>
-    </header>
-  </div>
+<div class="learn">
+  {# Hero starts #}
+  <header class="region">
+    <div class="wrapper">
+      {# Note: this needs replacing with the new headline pattern once the new homepage is merged #}
```

Comment:
Can this comment go away - we're merging the new homepage?

Reply:
> Here's some thoughts/issues
> 
> On `/measure`, I had to test it on the live site cause I think the key is blocked from the netlify preview, but the results are displayed in dark mode even though my experience is set in light mode. Not a breaking issue, but a wonky design thing.
> 
> ![image](https://user-images.githubusercontent.com/11811422/139790972-64d2e92d-c50d-4504-9d6f-f8237453f649.png)
> 
> Learn design's logo in mobile, or with the compressed toc is very wonky. Not a breaking issue.
> 
> https://deploy-preview-6615--web-dev-staging.netlify.app/learn/design/
> 
> ![image](https://user-images.githubusercontent.com/11811422/139791067-289ee9b2-9188-492a-8e91-473544368f09.png)
> 
> Nothing I think that should stop this from landing, just things to fix at some point.

Hey @jimoong, I reckon we're gonna need some one-line logo variants for this sort of context

Reply:
Maybe we should use text (just black) instead svg on the app bar on mobile? I can do whichever way easier but by using text we may have better control on edge cases like the longer title (such as ellipsis, etc.). WDYT?